playbooks/roles/deploy-osh/files/020-ingress.sh

```diff
@@ -17,7 +17,7 @@
 set -xe
 
 #NOTE: Deploy global ingress
-: ${OSH_INFRA_PATH:="../openstack-helm-infra"}
+: "${OSH_INFRA_PATH:="../openstack-helm-infra"}"
```

Comment:
I'm not a bash expert, but this has double quotes already inside it; will it cause any issues? (it works in my testing so I suspect it's fine)

Reply:
It's probably working because we're not entering that part of the default assignment, but I would be glad to see an answer.

Reply:
Im my testing it did work, which is something that surprised me, let me retest it just in case....

Reply:
it actually works (would need some bash expert to explain why to me) but it also works without the double quotes inside it, so Im gonna remove those